spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureCloudConfigWatch.java

```diff
@@ -186,11 +186,10 @@ public String getMessage() {
      * @return the full name of the key mapping to the configuration store
      */
     private String watchedKeyNames(ConfigStore store, Map<String, List<String>> storeContextsMap) {
-        String prefix = store.getPrefix();
         String watchedKey = store.getWatchedKey().trim();
         List<String> contexts = storeContextsMap.get(store.getName());
 
-        String watchedKeys = contexts.stream().map(ctx -> genKey(prefix, ctx, watchedKey))
+        String watchedKeys = contexts.stream().map(ctx -> genKey(ctx, watchedKey))
```

Comment:
Why remove the prefix？

Reply:
The issues is that ctx already has the prefix attached to it, so it resulted in /<prefix>/<prefix>/application>/